templates/deployment.yaml

```
@@ -0,0 +1,73 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: metadata-service
+  labels:
+    k8s-app: metadata-service
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      k8s-app: metadata-service
+  template:
+    metadata:
+      annotations:
+        fluentd.packet.net/efk: stdout+stderr
+        fluentd.packet.net/efk-index: metadata-service
+      labels:
+        k8s-app: metadata-service
+    spec:
+      containers:
+        - name: metadata-service
+          env:
+          - name: API_URL
+            value: '{{ .Values.packetAPI }}'
+          - name: METADATASERVICE_OIDC_AUDIENCE
+            value: "{{ .Values.oidc.audience }}"
+          - name: METADATASERVICE_OIDC_ISSUER
+            value: "{{ .Values.oidc.issuer }}"
+          - name: METADATASERVICE_OIDC_JWKSURI
+            value: "{{ .Values.oidc.jwksuri }}"
+          - name: METADATASERVICE_OIDC_CLAIMS_ROLES
+            value: "{{ .Values.oidc.rolesClaim }}"
+          - name: METADATASERVICE_OIDC_CLAIMS_USERNAME
+            value: "{{ .Values.oidc.userClaim }}"
+          - name: METADATASERVICE_DB_URI
+            valueFrom:
+              secretKeyRef:
+                name: {{ template "common.names.fullname" . }}-dbconn
+                key: uri
+          image: "{{ .Values.metadataservice.image.repository }}:{{ .Values.metadataservice.image.tag }}"
+          imagePullPolicy: Always
+          volumeMounts:
+          - name: dbcerts
+            mountPath: "/dbcerts"
+            readOnly: true
+          ports:
+            - name: http
+              containerPort: 9090
+              protocol: TCP
+          livenessProbe:
+            httpGet:
+              path: /_packet/healthcheck
+              port: http
+            initialDelaySeconds: 5
+            timeoutSeconds: 2
+          readinessProbe:
+            httpGet:
+              path: /healthcheck
+              port: http
+            initialDelaySeconds: 5
+            timeoutSeconds: 2
+          resources:
+            limits:
+              cpu: 4
+              memory: 4Gi
+            requests:
+              cpu: 4
+              memory: 4Gi
```

I would define the actual limits/requests in `values.yaml` instead of directly here 
```suggestion
        resources:
{{ toYaml .Values.resources | indent 10 }}
```

---

Ah, I noticed this pattern was used in the serverservice helm chart. Fixed.